proxy/src/lib.rs

```diff
@@ -29,8 +29,11 @@ extern crate prost_types;
 extern crate quickcheck;
 extern crate rand;
 extern crate tokio_connect;
-extern crate tokio_core;
-extern crate tokio_io;
+extern crate tokio;
```

Comment:
Do we need the other tokio crates besides just `tokio`?

Reply:
`tokio_connect` is not part of the main `tokio` crate yet.

Reply:
@carllerche I think @seanmonstar is referring to the `extern crate tokio_reactor/tokio_threadpool/tokio_timer` that were previously here when I was manually constructing the runtimes; those are no longer required.